### ISSUE TITLE
stuck 

### ISSUE THREAD

**Issue by klyxmaster** [gibbed/SteamAchievementManager#202]
stuck on RETRIEVING STAT INFORMATION...

**Comment by ToadCast** [comment]
I hope you do not expect any help with this kind of comment.

**Comment by klyxmaster** [comment]
Nope, figured it out,  pretty basic question (comment), although I suppose a screnny would have helped clarify it.  To those that might have the same issue, make sure your not running ARCHI in the background or on another computer. That's what wired for me.

**Comment by ToadCast** [comment]
Since the first message has close to zero information, anything could be good to help someone else in the future.
If it's ok on your side, you can close the issue, thanks.

**Comment by savasibo97** [comment]
Could you please tell me what ARCHI is? Because currently I have the same problem. The program opens up normally, I select a game whether it's installed or not, then it gets stuck with the text "Retrieving stat information" at the bottom. I don't know what I did wrong, I didn't even tamper with any files packed inside the zip file.

**Comment by klyxmaster** [comment]
Yes, it is a steam idler called Archi Steam Farm, used to idle for cards. In short, make sure there is nothing using your steam when you use SAM.
